packages/node_modules/pouchdb-core/src/adapter.js

```diff
@@ -403,7 +403,7 @@ class AbstractPouchDB extends EventEmitter {
         });
       }
 
-      ids.map(function (id) {
+      ids.forEach(function (id) {
```

Comment:
In my opinion, a for ... of loop suites this block better:
because `this` is bound to `this` of the outer scope.

Reply:
Alt.: an arrow-fn

Reply:
:+1: both nicer options, but I want to keep incidental changes minimal.